frontend/packages/core/src/AppLayout/stories/searchfield.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from "react";
+import { MemoryRouter } from "react-router";
+import type { Meta } from "@storybook/react";
+
+import { ApplicationContext } from "../../Contexts/app-context";
+import SearchField from "../search";
+import TerminateInstance from "../../../../../workflows/ec2/src/terminate-instance"
+import ResizeAutoscalingGroup from "../../../../../workflows/ec2/src/resize-asg";
+
+export default {
+  title: "Core/AppLayout/Search Field",
+  component: SearchField,
+  decorators: [
+    () => (
+      <MemoryRouter>
+        <SearchField />
+      </MemoryRouter>
+    ),
+    StoryFn => {
+      return (
+        <ApplicationContext.Provider value={{ workflows: [
```

Comment:
perhaps something to follow up on is modifying the search field to take an interface